plugins/repository-hdfs/build.gradle

```diff
@@ -57,6 +57,10 @@ configurations {
   hdfsFixture
 }
 
+configurations.hdfsFixture {
+    exclude group: "commons-io", module: "commons-io"
```

Comment:
I think we should apply it to artifacts:
```
api("org.apache.hadoop:hadoop-hdfs:${versions.hadoop3}") {
    exclude group: "commons-io", module: "commons-io"
}
```

Reply:
I tried that, it didn't work. It's brought in by this fixture. See https://advisories.opensearch.org/advisories/CVE-2024-47554

I've confirmed this fixes the issue.

Reply:
It's either directly from the fixture or from the minicluster dependency brought in by the fixture.
```
test/fixtures/hdfs-fixture/build.gradle compileClasspath org.apache.hadoop-hadoop-minicluster@3.3.5 org.apache.hadoop-hadoop-hdfs@3.3.5 commons-io-commons-io@2.11.0
```

Reply:
Ah, figured it out, there's a whole other `build.gradle` for test/fixtures/hdfs-fixture.   Added the version and excluded it from the dependencies.  It works....

```
✗ ./gradlew :plugins:repository-hdfs:dependencies | grep commons-io
+--- commons-io:commons-io:2.16.0 (n)
+--- commons-io:commons-io:2.16.0
     +--- commons-io:commons-io:2.16.0
     |    |    |    |    \--- commons-io:commons-io:2.11.0 -> 2.16.0
+--- commons-io:commons-io:2.16.0
+--- commons-io:commons-io:2.16.0
+--- commons-io:commons-io:2.16.0
```